src/@types/Form.d.ts

```diff
@@ -2,7 +2,7 @@ export interface FormFieldProps {
   label: string
   name: string
   type?: string
-  options?: string[]
+  options?: any[]
```

Comment:
in the `AssetSelection` component I created the `AssetSelectionAsset` typing. So we should be able to use `string[]` | `AssetSelectionAsset[]` here instead of `any`